staging/src/k8s.io/apiserver/pkg/audit/evaluator.go

```diff
@@ -36,10 +36,24 @@ type AuditContext struct {
 // RequestAuditConfig is the evaluated audit configuration that is applicable to
 // a given request. PolicyRuleEvaluator evaluates the audit policy against the
 // authorizer attributes and returns a RequestAuditConfig that applies to the request.
-type RequestAuditConfig struct{}
+type RequestAuditConfig struct {
+	// OmitStages is the stages that need to be omitted from being audited.
+	OmitStages []audit.Stage
```

Comment:
`OmitManagedFields` will be added here in https://github.com/kubernetes/kubernetes/pull/94986